_overviews/scala3-book/domain-modeling-oop.md

```diff
@@ -74,14 +145,27 @@ trait GreetingService:
 trait TranslationService:
   def translate(text: String): String = "..."
 ```
+
+{% endtab %}
+{% endtabs %}
+
 To compose the two services, we can simply create a new trait extending them:
+
+{% tabs traits_5 %}
+{% tab 'Scala 2 and 3' for=traits_5 %}
+
 ```scala
 trait ComposedService extends GreetingService, TranslationService
```

Comment:
`extends GreetingService, TranslationService` => `extends GreetingService with TranslationService` in scala 2

Reply:
corrected.